documentation/book/proc-bridge-retrieving-latest-messages-from-consumer.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-bridge-retrieving-latest-messages-from-consumer-{context}']
+= Retrieving the latest messages from a Kafka Bridge consumer
+
+Next, retrieve the latest messages from the Kafka Bridge consumer by requesting data from the link:https://strimzi.io/docs/bridge/latest/#_poll[records^] endpoint. In production use, HTTP clients can call this endpoint repeatedly (in a loop).
+
+.Procedure
+
+.  Produce some more messages to the Kafka Bridge consumer, as described in xref:proc-producing-messages-from-bridge-topics-partitions-{context}[Producing messages to topics and partitions].
+
+.  Retrieve those messages from the Kafka Bridge consumer:
++
+[source,curl,subs=attributes+]
+----
+curl -X GET http://localhost:8080/consumers/bridge-quickstart-consumer-group/instances/bridge-quickstart-consumer/records \
+  -H 'accept: application/vnd.kafka.json.v2+json'
+----
++
+If the request is successful, the Kafka Bridge returns an array of messages -- describing the topic name, key, value, partition, and offset -- in the response body, along with a `200` code. Messages are retrieved from the latest offset by default.
++
+[source,json,subs=attributes+]
+----
+HTTP/1.1 200 OK
+content-type: application/vnd.kafka.json.v2+json
+#...
+[
+  {
+    "topic":"bridge-quickstart-topic",
+    "key":"my-key",
+    "value":"sales-lead-0001",
+    "partition":0,
+    "offset":0
+  },
+  {
+    "topic":"bridge-quickstart-topic",
+    "key":null,
+    "value":"sales-lead-0003",
+    "partition":0,
+    "offset":1
+  },
+#...
+----
++
+NOTE: If an empty array is returned, produce more records to the consumer as described in xref:proc-producing-messages-from-bridge-topics-partitions-{context}[Producing messages to topics and partitions], and then try retrieving messages again. 
```

Comment:
Not sure this is the right place, but we need to mention that a first call on a new Kafka Bridge consumer returns an empty response. 
'After creating and subscribing to a Kafka Bridge consumer, a first GET request will return an empty response as the the poll starts a rebalancing process to assign partitions.'  Or something like this. Please check with @ppatierno

Reply:
Thanks. I added this to the procedure for Paolo to check.